packages/tools/bundleDefaultPlugins.ts

```diff
@@ -45,16 +48,15 @@ export async function extractPlugins(currentDir: string, defaultPluginDir: strin
 	}
 }
 
-export const downloadPlugins = async (localPluginsVersions: PluginAndVersion, defaultPluginsInfo: DefaultPluginsInfo, manifests: any): Promise<any> => {
+export const downloadPlugins = async (localPluginsVersions: PluginAndVersion, defaultPluginsInfo: DefaultPluginsInfo, manifests: any): Promise<PluginIdAndName> => {
 
 	const downloadedPluginsNames: PluginIdAndName = {};
 	for (const pluginId of Object.keys(defaultPluginsInfo)) {
 		if (localPluginsVersions[pluginId] === defaultPluginsInfo[pluginId].version) continue;
 		const response = await fetch(`https://registry.npmjs.org/${manifests[pluginId]._npm_package_name}`);
 
 		if (!response.ok) {
-			const responseText = await response.text();
-			throw new Error(`Cannot download plugin file ${responseText.substr(0,500)}`);
+			throw new Error(`Cannot fetch ${manifests[pluginId]._npm_package_name} release info from NPM`);
```

Comment:
Actually your previous code was good, as you were showing the actual error message. Any reason you removed it?

Reply:
Actually, while testing locally I thought that `.text()` will show buffer data but on failing to fetch it won't have any buffer data to show :sweat_smile: . Restored the previous message.